tools/eslint-config/angular.json

```diff
@@ -22,6 +21,7 @@
         "@angular-eslint/use-pipe-transform-interface": "error",
         "@angular-eslint/no-empty-lifecycle-method": "error",
         "@angular-eslint/no-output-native": "warn",
+        "@angular-eslint/prefer-standalone": "off",
```

Comment:
This rule is "auto-fixable" in eslint terms, but that means that if it is on, it is immediately removed from source code due to our "lint on save" functionality or when committing (because of lint-staged).

Some of our design system comps are not standalone, so it should definitely not remove the `standalone: false` flag from those 😬